src/game-of-life/hello-world.md

```diff
@@ -5,148 +5,337 @@ program: a Web page that alerts "Hello, World!"
 
 ## Clone the Project Template
 
-The project template contains a "hello world" program. It comes pre-configured
-with sane defaults, so you can quickly build, integrate, and package your code
-for the Web.
+The project template comes pre-configured with sane defaults, so you can quickly
+build, integrate, and package your code for the Web.
 
-Clone this tutorials code repository, enter its directory, and checkout the
-`chapter-zero` branch:
+Clone the project template with this command:
 
 ```text
-git clone https://github.com/rustwasm/wasm_game_of_life.git
-cd ./wasm_game_of_life
-git checkout -b chapter-zero origin/chapter-zero
+cargo generate --git https://github.com/rustwasm/wasm-pack-template
```

Comment:
Did we already tell people to install `cargo-generate`?

Reply:
Yes, the setup instructions says to install `cargo-generate`.